htdocs/comm/action/class/actioncomm.class.php

```diff
@@ -1721,6 +1721,13 @@ public function build_exportfile($format, $type, $cachedelay, $filename, $filter
                     }
                     $diff++;
                 }
+
+				$parameters=array('filters' => $filters, 'eventarray' => $eventarray);
```

Comment:
What do you think of changing
$parameters=array('filters' => $filters, 'eventarray' => $eventarray);
into
$parameters=array('filters' => $filters, 'eventarray' => &$eventarray);

$eventarray can be very large. But i am not sure it works

Reply:
I thought it was a good idea but to respect what is done everywhere else I decided not to do that.
I fix it now.